### ISSUE TITLE
mcfensi.com

### ISSUE THREAD

**Issue by sxjeru** [StopModReposts/Illegal-Mod-Sites#142]
[This site](http://www.mcfensi.com/) distributes a lot of mods without authorization from the modder. 

Not only that, the site has a huge amount of ADs. And it even forces players buy some good mods with money, but these mods are free on the CurseForge. 

The site is from China. I am a Chinese normal player. I want to do something for Minecraft Gamecircle of China. So please add it to Blacklist. 

BTW, the site is also distributing modpacks, texture packs, skins, etc. 

Thanks, very much.

**Comment by bilde2910** [comment]
Hi, @sxjeru and thanks for the submission. I tried visiting the site, but I couldn't find much evidence of advertisements, and I also couldn't immediately find people selling mods. That said, I couldn't check that much, because the site loads extremely slowly for me.

Can you send me a link to a page that sells mods, or that has a lot of ads on it? I would do more investigation myself, but each page takes 5 minutes to load.

Thanks for posting! ^.^

**Comment by sxjeru** [comment]
@bilde2910 Sorry, maybe my words were too extreme. The site has some ADs, they are on the left of the home page. In my view, they not particularly affect browsing. 

About selling mods, it usually use points of bbs to exchange mods, and the points can increase by recharge, there are also other ways to get points, just some difficult.

Of course the site are distributing mods, and I can't find anything about authorization. Such as [this](http://www.mcfensi.com/forum.php?mod=viewthread&tid=5404).

If you want to test download, it needs to be registered. I hope you can get some evidences.

Today, I have investigated this site for a long time. Here is my final recommended rating. (According to your Illegal Sites list, and it for reference only)
`1` `3` `2`

**Comment by bilde2910** [comment]
Sorry that this took so long. We had a Chinese speaker review the site and check the claims. We struggled to find ads and selling of mods. The admin makes most of the posts, and they're mostly descriptive. All download links I could find pointed to CurseForge, including the example you provided. We also found an email address for reporting reposts/mods in violation of their licenses.

For this reason, we will not add the site to the list at this time. Specific evidence of selling mods (such as screenshots, preferably with accompanying links to where they were taken, if possible) will help prove that claim and we can reconsider the case if needed, but for now, we do not have enough evidence to add it to the list based on review and the reasons you listed.
